spec/compiler/semantic/macro_spec.cr

```diff
@@ -1122,4 +1122,51 @@ describe "Semantic: macro" do
       Moo.bar
       ), inject_primitives: false) { int32 }
   end
+
+  it "passes #4739" do
```

Comment:
I think we should have meaningful spec description, mentioning the issue at the end of the description, if applicable:
`finds macro instead of method when inside a metaclass (#4739)` or something like that.